openshift/packageAnalyzerSyncJob-template.yaml

```diff
@@ -0,0 +1,161 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: graph-sync-job-package-analyzer
+  annotations:
+    description: "Thoth: Graph Sync Job for package-analyzer documents"
+    openshift.io/display-name: "Thoth: Graph Sync Job Package-Analyzer"
+    version: 0.6.1
+    tags: thoth,ai-stacks,aistacks,graph-sync-job
+    template.openshift.io/documentation-url: https://github.com/Thoth-Station/
+    template.openshift.io/long-description: >
+      This template defines resources needed to deploy Thoth's Graph Sync Job for Package-Analyzers on OpenShift.
+    template.openshift.io/provider-display-name: Red Hat, Inc.
+  labels:
+    template: graph-sync-job-package-analyzer
+    app: thoth
+    component: graph-sync
+    graph-sync-type: package-analyzer
+
+parameters:
+  - description: Registry the ImageStream to be use lives in
+    displayName: ImageStream Registry
+    required: true
+    name: IMAGE_STREAM_REGISTRY
+    value: "docker-registry.default.svc:5000"
+
+  - description: Namespace the ImageStream to be use lives in
+    displayName: ImageStream Project
+    required: true
+    name: IMAGE_STREAM_NAMESPACE
+    value: "thoth-infra-stage"
+
+  - description: Tag of the ImageStream to be use
+    displayName: ImageStream Tag
+    required: true
+    name: IMAGE_STREAM_TAG
+    value: "latest"
+
+  - name: THOTH_SYNC_DOCUMENT_ID
+    required: true
+    description: Document id to be synced.
+
+  - name: THOTH_JOB_ID
+    required: true
+    description: Graph sync job identifier.
+
+objects:
+  - apiVersion: batch/v1
+    kind: Job
+    metadata:
+      name: "${THOTH_JOB_ID}"
+      labels:
+        app: thoth
+        component: graph-sync
+        graph-sync-document-id: "${THOTH_SYNC_DOCUMENT_ID}"
+        graph-sync-type: package-analyzer
+        # Clean this up after one hour.
+        mark: cleanup
+        ttl: "1h"
+    spec:
+      backoffLimit: 10
+      template:
+        metadata:
+          labels:
+            app: thoth
+            component: graph-sync
+            graph-sync-document-id: "${THOTH_SYNC_DOCUMENT_ID}"
+            graph-sync-type: package-analyzer
+            mark: cleanup
+            ttl: "1h"
+        spec:
+          containers:
+            - name: graph-sync
+              image: "${IMAGE_STREAM_REGISTRY}/${IMAGE_STREAM_NAMESPACE}/graph-sync-job:${IMAGE_STREAM_TAG}"
+              env:
+                - name: THOTH_ONLY_PACKAGE_ANALYSIS_DOCUMENTS
```

Comment:
Can be the variable renamed to `THOTH_ONLY_PACKAGE_ANALYZER_DOCUMENTS` to be consistent?

Reply:
For sure, that is better, changed.